ci: broken links checker, test on push

B8-RX opened the pull request:
## Information

| Type                  | Breaking change |
| --------------------- | --------------- |
| Feature/Fix/Doc/Chore | Yes/No          |

---

<!--
## Usage example

Example to use your feature and to improve the documentation after merging your PR:
```typescript
import {} from "@tsed/common";

```
-->

## Todos

- [ ] Tests
- [ ] Coverage
- [ ] Example
- [ ] Documentation

B8-RX committed:
ci: broken links checker, test on push

B8-RX committed:
ci: broken links checker, test on push

B8-RX committed:
ci: broken links checker, test on push

B8-RX committed:
ci: broken links checker, test on push

B8-RX committed:
ci: broken links checker, test on push

i delete Target:  in BODY property

B8-RX committed:
ci: broken links checker, test on push

recursive set to false

Romakita commented:
@sofianeSghiouar Ce qu'on voit c'est que sans le récursive on fait juste le controle de la home page :/ C'est dommage. Faudrait trouver un moyen de faire un controle sur les pages de documentation et tutorials à minima.

Romakita commented:
Pour l'instant ca ne rempli pas les critères d'acceptances. Essai de voir si tu peux trouver une alternative plus rapide et complète. Il existe plusieurs outils qui check les liens morts sur un site avec github action ^^.

B8-RX committed:
ci: broken links checker, test on push

test link checker on repo

source: https://github.com/lycheeverse/lychee-action

B8-RX committed:
ci: broken links checker, test on push

test link checker on repo

source: https://github.com/lycheeverse/lychee-action

remove repository_dispatch and workflow_dispatch

B8-RX committed:
ci: broken links checker, test on push

test link checker on repo

source: https://github.com/lycheeverse/lychee-action

remove repository_dispatch and workflow_dispatch

Romakita commented:
Good job. Ca semble etre mieux non?

sofianeSghiouar commented:
> Good job. Ca semble etre mieux non?

B8-RX committed:
tech: set link checker to run every thursday at 11h30 am

B8-RX committed:
tech: set link checker to run every thursday at 12h10 pm

B8-RX committed:
tech: set link checker to run every thursday at 12h10 pm

B8-RX committed:
tech: set link checker to run every thursday at 12h30 pm

B8-RX committed:
tech: set link checker to run every thursday at 13h35 pm

B8-RX committed:
tech: set link checker to run every thursday at 13h45 pm

B8-RX committed:
tech: set link checker to run every sunday at 23h23

sofianeSghiouar commented:
The link checker is now configured to run once a week, i have configured it to run every Sunday at 11h23 PM :)

Romakita commented:
:tada: This PR is included in version 6.114.0 :tada:

The release is available on:
- `v6.114.0`
- [GitHub release](https://github.com/tsedio/tsed/releases/tag/v6.114.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
